ui/packages/catalog-ui-search/src/main/webapp/component/result-item/result-item.view.js

```diff
@@ -39,16 +40,53 @@ const HandleBarsHelpers = require('../../js/HandlebarsHelpers.js')
 const ResultLinkView = require('../result-link/result-link.view.js')
 const plugin = require('plugins/result-item')
 
+const LIST_DISPLAY_TYPE = 'List'
+const GRID_DISPLAY_TYPE = 'Grid'
+
+const renderResultLink = (shouldRender, model) =>
+  shouldRender && (
+    <MarionetteRegionContainer
+      className="result-link is-button is-neutral composed-button"
+      data-help="Follow external links."
+      view={props => PopoutView.createSimpleDropdown(props)}
```

Comment:
```suggestion
      view={PopoutView.createSimpleDropdown({
        componentToShow: ResultLinkView,
        modelForComponent: model,
        leftIcon: 'fa fa-external-link',
      })}
```

In this case we can instantiate the view and pass it in.

Reply:
This would produce the same result, but would result in a little more eager view instantiation, no? I did this specifically to delay view creation till right before it was needed.

Reply:
Here's a video with reproduction instructions for the bug: 
https://www.youtube.com/watch?v=n8-c8oOmV4k&feature=youtu.be

The short story is that you have to have a result with external links and run a search while in gallery mode.  The external links dropdown will have issues rendering correctly.  This doesn't happen if you are in list mode when the search is initially run.  

If the PR is rebased, this will start working. (see https://github.com/codice/ddf/pull/4412)  I demonstrate this in the video by cherry-picking in the commit that fixes this.  So I think once this is rebased we're good to go ahead and merge it.

Reply:
@andrewkfiedler I rebased against master. The error doesn't show up in either gallery or list view.